SignalServiceKit/src/Storage/TSStorageManager.m

```diff
@@ -187,10 +187,23 @@ @implementation OWSUnarchiverDelegate
 
 - (nullable Class)unarchiver:(NSKeyedUnarchiver *)unarchiver cannotDecodeObjectOfClassName:(NSString *)name originalClasses:(NSArray<NSString *> *)classNames
 {
+    DDLogVerbose(@"%@ Could not decode object: %@", self.tag, name);
```

Comment:
Might as well make this DDLogError, as the details would be helpful in production.

Reply:
Done.